Makefile

```diff
@@ -264,7 +248,7 @@ deploy-onprem:
 	podman pod create --name assisted-installer -p 5432,8000,8090,8080
 	# These are required because when running on RHCOS livecd, the coreos-installer binary and
 	# livecd are bind-mounted from the host into the assisted-service container at runtime.
-	[ -f livecd.iso ] || curl $(BASE_OS_IMAGE) --retry 5 -o livecd.iso
+	[ -f livecd.iso ] || until curl https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/latest/rhcos-live.x86_64.iso --retry 5 -o livecd.iso; do sleep 1; done
```

Comment:
why hard-code the URL?

Reply:
Resolving this in favor of https://github.com/openshift/assisted-service/pull/645#discussion_r538139450